packages/nx/src/utils/ignore.ts

```diff
@@ -0,0 +1,151 @@
+import * as fastGlob from 'fast-glob';
+import * as fs from 'fs';
+import ignore, { Ignore } from 'ignore';
+import * as path from 'path';
+import { Tree } from '../generators/tree';
+import { workspaceRoot } from './workspace-root';
+import { normalizePath } from './path';
+
+const workspaceIgnoreFiles = ['.gitignore', '.nxignore'];
+
+export function createWorkspaceIgnore(root = workspaceRoot): Ignore {
+  return createIgnoreFromFS(root, workspaceIgnoreFiles);
+}
```

Comment:
This doesn't seem needed. Swap the params on `createIgnoreFromFS` to have default values and this function can be removed without worsening DX